src/AppBundle/DataFixtures/ORM/LoadTodoItemData.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace AppBundle\DataFixtures\ORM;
+
+use DateTime;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use AppBundle\Entity\TodoItem;
+use function GuzzleHttp\Promise\iter_for;
+
+class LoadTodoItemData extends AbstractFixture implements OrderedFixtureInterface
+{
+    public function load(ObjectManager $manager)
+    {
+        $todoItem1 = new TodoItem();
+        $todoItem1->setCreatedAt(new DateTime('2018-01-01'));
```

Comment:
Kunne vi lagt inn noen todoitems som ikke er hardkodet for 2018 og? 
Tror dette skal sette `createdAt` til nåværende dato:
```php
$todoItem1->setCreatedAt(new DateTime());
```
Hvis de må være satt til første januar kan du f.eks hente ut nåværende år med `date('Y')` og ellers gjøre som før.